pkg/core/mempool/mem_pool.go

```diff
@@ -197,10 +218,63 @@ func (mp *Pool) Add(t *transaction.Transaction, fee Feer) error {
 	return nil
 }
 
+// checkAttributesConflicts is an internal unlocked method which returns an error in case
+// if transaction can not be added due to Conflicts attributes. It removes all conflicting
+// transactions from mempool in case if transaction has no conflicts or wins all conflicts.
+func (mp *Pool) checkAttributesConflicts(t *transaction.Transaction, fee Feer) error {
+	if fee.P2PSigExtensionsEnabled() {
+		var (
+			conflictingTxs []util.Uint256
+			ok             bool
+		)
+		// Step 1: check if `t` was in attributes of mempooled transactions
+		if conflictingTxs, ok = mp.conflicts[t.Hash()]; ok {
+			// check firstly if there are some mempooled transactions which have better network fee,
+			// if so, reject the new transaction
+			for _, hash := range conflictingTxs {
+				conflictingTx := mp.verifiedMap[hash]
+				if conflictingTx.HasSigner(t.Sender()) && conflictingTx.NetworkFee >= t.NetworkFee {
+					return fmt.Errorf("%w: conflicting transaction %s has bigger network fee", ErrConflictsAttribute, conflictingTx.Hash().StringBE())
+				}
+			}
+		}
+		// Step 2: check if mempooled transactions were in `t`'s attributes
+		for _, attr := range t.GetAttributes(transaction.ConflictsT) {
+			hash := attr.Value.(*transaction.Conflicts).Hash
+			existingTx, ok := mp.verifiedMap[hash]
+			if !ok {
+				continue
+			}
+			if !t.HasSigner(existingTx.Sender()) {
+				return fmt.Errorf("%w: not signed by the sender of conflicting transaction %s", ErrConflictsAttribute, existingTx.Hash().StringBE())
+			}
+			if existingTx.NetworkFee >= t.NetworkFee {
```

Comment:
And here.

Reply:
Actually, here `>=` is right. If fees are equal, the one without the attribute wins.